app/components/ch/components/DownloadList.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <li>
+    <button
```

Comment:
To be double checked, but looks like it's better to use a link with `download` attribute in place of a button element:
https://css-tricks.com/building-good-download-button/

Reply:
done and done